### DISCUSSION TITLE
my2sql插件配置后无法解析binlog内容

### DISCUSSION THREAD

**Discussion by cqvienet** [hhyo/Archery#1700] [General]
my2sql插件配置后无法解析binlog内容，后台nginx日志也没有任何内容，但是通过my2sql手工解析binlog没有问题
![image](https://user-images.githubusercontent.com/5274488/180390216-757c5f52-5fec-4b6d-8d8c-b66a69b2f835.png)

**Comment by nick2wang** [comment]
没有选择终止解析文件和位置时会有问题，可以参考这个pr更新下代码 #1653

**Reply by cqvienet** [reply]
> 没有选择终止解析文件和位置时会有问题，可以参考这个pr更新下代码 #1653

我将此文件更新后，重新docker,并配置了终止解析文件和位置还是解析不出内容，麻烦问一下还可能是什么其它的的原因？

**Reply by nick2wang** [reply]
选一个大一点的binlog文件解析看看，截图里文件大小只有541估计没有可解析的dml

**Reply by cqvienet** [reply]
> 选一个大一点的binlog文件解析看看，截图里文件大小只有541估计没有可解析的dml

我就造了一条数据，在本地是可以解析出来并生成rollback语句的

**Reply by cqvienet** [reply]
> 我就造了一条数据，在本地是可以解析出来并生成rollback语句的

刚刚批量插了一数据，还是一样的

**Reply by nick2wang** [reply]
看下archery日志
docker logs archery -f --tail=100

**Reply by cqvienet** [reply]
> 看下archery日志 docker logs archery -f --tail=100
刚刚看了访问my2sql这个页面会报如下错误
![archery_error](https://user-images.githubusercontent.com/5274488/180404720-e86e8a21-ecf7-4cde-a362-9d30e30e142e.png)

**Reply by nick2wang** [reply]
这个报错没关系，是redis配置的

**Reply by cqvienet** [reply]
> 这个报错没关系，是redis配置的

您好！可以麻烦再看一下我这个问题，是否可以加一下QQ或者微信沟通一下！

**Reply by cqvienet** [reply]
> 这个报错没关系，是redis配置的

nginx从登录页面跳转到主页面报以下错误
/opt/archery/static/metisMenu/js/metisMenu.min.js.map" failed (2: No such file or directory), client: 192.168.1.100, server: archery, request: "GET /static/metisMenu/js/metisMenu.min.js.map
不知道这个对my2sql有不有影响？

**Reply by cqvienet** [reply]
> 不知道这个对my2sql有不有影响？

这个目录下只有这两个文件 metisMenu.min.34b1c6a35b5c.js  metisMenu.min.js

**Comment by nick2wang** [comment]
经排查是由于mysql密码中包含!(感叹号)导致命令行认证失败，已处理 #1709
